### PR TITLE
Merge bwagreement db into satellite master db

### PR DESCRIPTION
**Pull request by mniewrzal**
https://storjlabs.atlassian.net/browse/V3-849

### CODE REVIEW

File: satellite/satellitedb/bandwidthagreement.go [hunk 1]
```diff
@@ -24,36 +24,34 @@ func (b *bandwidthagreement) CreateAgreement(ctx context.Context, agreement *bwa
...
-	var agreements []*bwagreement.Agreement
-	for _, entry := range rows {
-		agreements = append(agreements, &bwagreement.Agreement{
-			Signature: entry.Signature,
-			Agreement: entry.Data,
-			CreatedAt: entry.CreatedAt,
-		})
+	agreements := make([]bwagreement.Agreement, len(rows))
+	for i, entry := range rows {
+		agreement := agreements[i]
```
Comment (egonelbre) [inline satellite/satellitedb/bandwidthagreement.go]: You need to take the pointer, otherwise you won't change the actual value in the array.

Comment (mniewrzal) [reply satellite/satellitedb/bandwidthagreement.go]: done

File: pkg/accounting/tally/tally_test.go [hunk 2]
```diff
@@ -61,11 +63,11 @@ func TestOnlineNodes(t *testing.T) {
 	assert.NoError(t, err)
 	defer func() { _ = accountingDb.Close() }()
 
-	bwDb, err := dbManager.NewDBManager("sqlite3", "file::memory:?mode=memory&cache=shared")
+	masterDB, err := satellitedb.NewDB("sqlite3://file::memory:?mode=memory&cache=shared")
 	assert.NoError(t, err)
-	defer func() { _ = accountingDb.Close() }()
+	defer func() { _ = masterDB.Close() }()
```
Comment (egonelbre) [inline pkg/accounting/tally/tally_test.go]: Don't ignore the errors.

Comment (mniewrzal) [reply pkg/accounting/tally/tally_test.go]: I will reorganize tests in next pr

**Pull request merged**